src/Command/Environment/AlignCommand.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CoreBundle\Command\Environment;
+
+use EMS\CommonBundle\Common\Command\AbstractCommand;
+use EMS\CommonBundle\Common\Standard\Json;
+use EMS\CommonBundle\Search\Search;
+use EMS\CommonBundle\Service\ElasticaService;
+use EMS\CoreBundle\Entity\Environment;
+use EMS\CoreBundle\Service\EnvironmentService;
+use EMS\CoreBundle\Service\PublishService;
+use EMS\CoreBundle\Service\Revision\RevisionService;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class AlignCommand extends AbstractCommand
+{
+    private RevisionService $revisionService;
+    private LoggerInterface $logger;
+    private EnvironmentService $environmentService;
+    private PublishService $publishService;
+    private ElasticaService $elasticaService;
+
+    private Environment $source;
+    private Environment $target;
+
+    private int $scrollSize;
+    private string $scrollTimeout;
+    private string $searchQuery;
+
+    public const ARGUMENT_SOURCE = 'source';
+    public const ARGUMENT_TARGET = 'target';
+    public const ARGUMENT_SCROLL_SIZE = 'scrollSize';
+    public const ARGUMENT_SCROLL_TIMEOUT = 'scrollTimeout';
+    public const OPTION_FORCE = 'force';
+    public const OPTION_SEARCH_QUERY = 'searchQuery';
+    public const OPTION_SNAPSHOT = 'snapshot';
+    public const DEFAULT_SCROLL_SIZE = 100;
```

Comment:
You should use the default bulk size (env variables). See other commands

Reply:
Add a revisionSearcher service, he is now aware of the default scroll size